pkg/reconciler/containersource/resources/labels.go

```diff
@@ -24,5 +24,7 @@ func Labels(name string) map[string]string {
 	return map[string]string{
 		"sources.knative.dev/source":          containerSourceController,
 		"sources.knative.dev/containerSource": name,
+		"app.kubernetes.io/name":              "knative-eventing",
+		"app.kubernetes.io/component":         "containersource-adapter",
```

Comment:
do we need to patch/edit the `pingsource` labels too?

Reply:
PingSource has them already

https://github.com/knative/eventing/blob/843b6f2d54376fee41fa3f450f862106da552f41/config/core/deployments/pingsource-mt-adapter.yaml#L22-L24